---
# Documentation: https://wowchemy.com/docs/managing-content/

title: PATHWAY-BASED ANALYSIS FOR GENOME-WIDE ASSOCIATION STUDIES USING SUPERVISED
  PRINCIPAL COMPONENTS
subtitle: ''
summary: ''
authors:
- Xi Chen
- Lily Wang
- Bo Hu
- Mingsheng Guo
- John Barnard
- Xiaofeng Zhu
tags: []
categories: []
date: '2010-11-01'
lastmod: 2022-04-14T10:57:42-04:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-04-14T14:57:42.164799Z'
publication_types:
- '2'
abstract: Many complex diseases are influenced by genetic variations in multiple genes,
  each with only a small marginal effect on disease susceptibility. Pathway analysis,
  which identifies biological pathways associated with disease outcome, has become
  increasingly popular for genome-wide association studies (GWAS). In addition to
  combining weak signals from a number of SNPs in the same pathway, results from pathway
  analysis also shed light on the biological processes underlying disease. We propose
  a new pathway-based analysis method for GWAS, the supervised principal component
  analysis (SPCA) model. In the proposed SPCA model, a selected subset of SNPs most
  associated with disease outcome is used to estimate the latent variable for a pathway.
  The estimated latent variable for each pathway is an optimal linear combination
  of a selected subset of SNPs; therefore, the proposed SPCA model provides the ability
  to borrow strength across the SNPs in a pathway. In addition to identifying pathways
  associated with disease outcome, SPCA also carries out additional within-category
  selection to identify the most important SNPs within each gene set. The proposed
  model operates in a well-established statistical framework and can handle design
  information such as covariate adjustment and matching information in GWAS. We compare
  the proposed method with currently available methods using data with realistic linkage
  disequilibrium structures and we illustrate the SPCA method using the Wellcome Trust
  Case-Control Consortium Crohn Disease (CD) dataset.
publication: '*Genetic epidemiology*'
doi: 10.1002/gepi.20532
links:
- name: URL
  url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480088/
---
